packages/kbn-pr-utils/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@kbn/pr-utils",
+  "main": "./target/index.js",
+  "types": "./target/index.d.ts",
+  "version": "1.0.0",
+  "license": "SSPL-1.0 OR Elastic License 2.0",
+  "private": true,
+  "scripts": {
+    "build": "../../node_modules/.bin/tsc",
+    "kbn:bootstrap": "yarn build",
```

Comment:
Should these be using the bazel commands?

Reply:
Yeah, copied them from another package but I should remove them